src/Data/Scripts/DBData.sql

```diff
@@ -0,0 +1,105 @@
+CREATE TABLE IF NOT EXISTS category (
+    id UUID PRIMARY KEY,
+    rol VARCHAR NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS localization (
+    id UUID PRIMARY KEY,
+    name varchar(100),
+    geom GEOMETRY(Point, 4326)
+);
+
+CREATE TABLE IF NOT EXISTS specialty (
+    id UUID PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    user_client_id UUID NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS workSchedule (
+    id UUID PRIMARY KEY,
+    startTime VARCHAR NOT NULL,
+    endTime VARCHAR NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS client (
+    id UUID PRIMARY KEY,
+    localization_id UUID NOT NULL,
+    name VARCHAR NOT NULL,
+    emailAddress VARCHAR NOT NULL,
+    categoryId UUID NOT NULL,
+    workScheduleId UUID NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS jobs (
+    id UUID PRIMARY KEY,
+    user_client_id UUID NOT NULL,
```

Comment:
I think that is not necessary why is the reason to have a Client Id

Reply:
The client Id is required on the jobs data base

Reply:
ohhh okey thanks my bada